src/net/sf/memoranda/EventsManager.java

```diff
@@ -161,6 +161,28 @@ public static Collection getEventsForMonth(CalendarDate date) {
 		return monthEvents;
 	}
 	
```

Comment:
Should look like this:

**/
* Takes the current selected calendar day and makes a collection of all events
* of that day and following custom day range. US-101.
*
* @param date A date to start from
* @param dayRange An int specifying the number of days in the range following the date
* @return A vector of dates within the range
*/